src/dbcp/metadata/data_mart.py

```diff
@@ -529,64 +527,46 @@
     schema=schema,
 )
 
-iso_counties_new_clean_n_projects_changelog = Table(
-    "iso_counties_new_clean_n_projects_changelog",
+counties_active_projects_capacity_mw_change_log = Table(
+    "counties_active_projects_capacity_mw_change_log",
     metadata,
-    Column("county_id_fips", String),
-    Column("2022-03", Float),
-    Column("2022-06", Float),
-    Column("2022-09", Float),
-    Column("2022-12", Float),
-    Column("2023-03", Float),
-    Column("2023-06", Float),
-    Column("2023-09", Float),
-    Column("2023-12", Float),
-    Column("2024-03", Float),
+    Column("county_id_fips", String, primary_key=True),
+    Column("report_date", DateTime, primary_key=True),
+    Column("clean_capacity_mw", Float),
+    Column("fossil_capacity_mw", Float),
+    Column("other_capacity_mw", Float),
     schema=schema,
 )
-iso_counties_new_clean_capacity_mw_changelog = Table(
-    "iso_counties_new_clean_capacity_mw_changelog",
+
+iso_regions_active_projects_capacity_mw_change_log = Table(
```

Comment:
Do these tables need documentation or are they self-explanatory?

Reply:
Whoops! I totally wrote documentation but I forgot to push it!